crmd/remote_lrmd_ra.c

```diff
@@ -662,9 +685,14 @@ handle_remote_ra_stop(lrm_state_t * lrm_state, remote_ra_cmd_t * cmd)
     }
     ra_data->cmds = NULL;
     ra_data->recurring_cmds = NULL;
-    ra_data->cur_cmd = NULL;
 
-    report_remote_ra_result(cmd);
+    if (cmd) {
+        cmd->rc = PCMK_OCF_OK;
+        cmd->op_status = PCMK_LRM_OP_DONE;
+        ra_data->cur_cmd = NULL;
```

Comment:
Probably cleaner although handle_remote_ra_stop is called without cmd (the new case) just
if cur_cmd == NULL.
So in the new call hierarchy it is actually assured that cur_cmd == NULL so I didn't think
much about what would happen if it wasn't as this is just true in the cases which have been
there already.
The command-list is freed and the sources added to the mainloop by the commands are
removed from the mainloop.
I think question should rather be if we should catch errors that might occur when there
is a command pending so that remote_lrm_op_callback doesn't branch into the new
stopping code.